src/main/java/intellistart/interviewplanning/model/interviewerslot/InterviewerSlotDtoValidator.java

```diff
@@ -111,12 +97,10 @@ public InterviewerSlot validateAndCreate(InterviewerSlotDtoRequest interviewerSl
    *
    * @throws UserException - when invalid interviewer id, role not Interviewer
    */
-  public InterviewerSlot validateAndUpdate(InterviewerSlotDtoRequest interviewerSlotDto,
-      Authentication authentication, Long userId, Long slotId)
+  public void validateUpdating(InterviewerSlot interviewerSlot,
+                                          Authentication authentication, Long userId, Long slotId)
```

Comment:
strange formatting

Reply:
This class in will be rewritten to Kotlin, so I didn't focus much on formatting of all java classes that will be rewritten